org.eclipse.scanning.test/src/org/eclipse/scanning/test/command/ScanRequestCreationTest.java

```diff
@@ -189,7 +282,7 @@ public void testGridCommandWithROINoDetector() throws Exception {
 	}
 
 	@Test
-	public void testStepCommandWithMonitors() throws Exception {
+	public void testStepCommandWithMonitors() {
 		pi.exec("sr =                               "
 			+	"scan_request(                      "
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "scan_request(                      " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)